### DISCUSSION TITLE
Adding a title to a graph

### DISCUSSION THREAD

**Discussion by bmesuere** [observablehq/plot#1029] [Q&A] [answered]
I'm a bit embarrassed to ask, but is there a way of adding a title to a graph?

I haven't seen any examples with a title and I haven't found any reference in the API documentation, but since captions are supported it seems like titles should be as well?

**Comment by mbostock** [comment, marked as answer]
It’s not implemented yet but you can thumbs-up this issue: https://github.com/observablehq/plot/issues/92
